### PR TITLE
Port System.Diagnostics.Metrics new docs (second pass)

### PR DESCRIPTION
**Pull request by carlossanlop**
Ports some APIs missed in the previous pass. cc @dotnet/area-system-diagnostics-metric 

Contributes to https://github.com/dotnet/runtime/issues/105980

### CODE REVIEW

**Comment by carlossanlop** [comment]
@tarekgh @noahfalk @tommcdon can you please give me a code review?

File: xml/System.Diagnostics.Metrics/InstrumentAdvice`1.xml [hunk 1]
```diff
@@ -67,9 +67,9 @@
         <ReturnType>System.Collections.Generic.IReadOnlyList&lt;T&gt;</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Gets the explicit bucket boundaries advised to be used with histogram instruments.</summary>
         <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <remarks>&lt;para&gt;Notes:&lt;/para&gt;&lt;para&gt;&lt;list type="bullet"&gt;&lt;/para&gt;&lt;para&gt;&lt;item&gt;A &lt;see langword="null" /&gt; value means no bucket boundaries have been configured and default values should be used for bucket configuration.&lt;/item&gt;&lt;/para&gt;&lt;para&gt;&lt;item&gt;An empty set of bucket boundaries hints that the histogram by default should NOT contain buckets and should only track count and sum values.&lt;/item&gt;&lt;/para&gt;&lt;para&gt;&lt;item&gt;A set of distinct increasing values for bucket boundaries hints that the histogram should use those for its default bucket configuration.&lt;/item&gt;&lt;/para&gt;&lt;para&gt;&lt;/list&gt;&lt;/para&gt;</remarks>
```
Comment (tarekgh) [inline xml/System.Diagnostics.Metrics/InstrumentAdvice`1.xml:72]: This looks wrong:

image

**Review by tarekgh — APPROVED** [review]
Added a comment. LGTM otherwise.

File: xml/System.Diagnostics.Metrics/InstrumentAdvice`1.xml (hunk 1 above)
Comment (carlossanlop) [reply xml/System.Diagnostics.Metrics/InstrumentAdvice`1.xml:72]: I don't know what happened there 😖

File: xml/System.Diagnostics.Metrics/InstrumentAdvice`1.xml [hunk 2]
```diff
@@ -1,10 +1,10 @@
-<Type Name="InstrumentAdvice&lt;T&gt;" FullName="System.Diagnostics.Metrics.InstrumentAdvice&lt;T&gt;">
```
Comment (carlossanlop) [inline xml/System.Diagnostics.Metrics/InstrumentAdvice`1.xml:1]: Agh I messed up. Need to revert this.

File: xml/System.Diagnostics.Metrics/InstrumentAdvice`1.xml (hunk 1 above)
Comment (tarekgh) [reply xml/System.Diagnostics.Metrics/InstrumentAdvice`1.xml]: Looks the tool cannot handle porting such format?

https://github.com/dotnet/runtime/blob/62784822a187d734cc13b595aaadc315e1c915e0/src/libraries/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Metrics/InstrumentAdvice.cs#L28

**Review by noahfalk — APPROVED** [review]
The text looks fine aside from presumably needing escape sequences for some of special characters

**Pull request merged**